hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/bootstrap/BootstrapOperator.java

```diff
@@ -227,7 +228,10 @@ protected void loadRecords(String partitionPath) throws Exception {
             .filter(logFile -> isValidFile(logFile.getFileStatus()))
             .map(logFile -> logFile.getPath().toString())
             .collect(toList());
-        HoodieMergedLogRecordScanner scanner = FormatUtils.logScanner(logPaths, schema, latestCommitTime.get().getTimestamp(),
+        InternalSchema internalSchema = new TableSchemaResolver(this.hoodieTable.getMetaClient())
```

Comment:
we can reuse the TableSchemaResolver which create in 200 line.

Reply:
Fixed